engine/common/host.c

```diff
@@ -902,8 +906,16 @@ int EXPORT Host_Main( const char *progname, int bChangeGame, pfnChangeGame func
 		}
 		else
 		{
-			strcat(cmdLine, arg);
-			strcat(cmdLine, " ");
+			#ifdef PANDORA
+			if( !strcmp( arg, "--noshouldermb" )) {
+				noshouldermb=1;
+			} else {
+			#endif
+				strcat(cmdLine, arg);
+				strcat(cmdLine, " ");
+			#ifdef PANDORA
+			}
+			#endif

```

Comment:
It will be better if define noshouldermb as cvar and set by Sys_GetParmFromCmdLine() function.